app/javascript/src/javascripts/notes.js

```diff
@@ -134,7 +134,9 @@ class Note {
 
     copy_style_attributes() {
       let $note_box = this.$note_box;
-      let $attribute_child = $note_box.find('.note-box-attributes');
+      let attributes_body = this.note.preview_body || "";
+      let $attributes_note = $('<div>' + attributes_body + '</div>');
```

Comment:
This contains an XSS bug. If you put `<img src="https://evil.com/tracker.gif" onerror="alert('xss')">` in an embedded note then it will execute malicious Javascript for anyone viewing the post.

Not sure what `preview_body` is for but it should come from the sanitized HTML returned by the `/note_previews` call.

Reply:
`/note_previews` currently strips out the z-index attribute. I think I can probably use a regex to safely harvest the z-index attribute though instead.